docs/source/conf.py

```diff
@@ -1,9 +1,14 @@
 import os
 import sys
-from pyro import __version__
 
 import sphinx_rtd_theme
 
+if 'READTHEDOCS' not in os.environ:
+    # install pyro on rtd so it can be imported
```

Comment:
Why does this say "on rtd" when the above conditional says "if not on rtd"?

Reply:
shoot thats a bug, good catch. double checked it against the tested branch